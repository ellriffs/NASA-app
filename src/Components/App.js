
import React, { useState } from 'react';
import logo from '../Assets/nasa-worm-logo.png'
import "../Styles/App.css";
import Search from './Search';
import SearchResults from './SearchResults'


function App() {
  const [searchResults, setSearchResults] = useState([])
  
  return (
    <div className="App">
      <img className="Logo" src={logo} alt="logo" />
      <Search 
      setSearchResults = {setSearchResults}
      />
      <SearchResults
      results={searchResults} 
      />
    </div>
  );
};

export default App;
