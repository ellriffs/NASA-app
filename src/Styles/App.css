


.Logo{
    display: flex;
    flex-direction: row;
    justify-content: center;
    height: 20rem;
    width:40rem;
    margin: auto;
    color-adjust: initial;
}