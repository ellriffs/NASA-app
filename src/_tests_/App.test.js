import { render } from '@testing-library/react';
import App from '../Components/App';

describe("App", () => {
    const { asFragment } = render(<App />)

    it('mounts App component to the virtual DOM', () => {
        expect(asFragment()).toMatchSnapshot();
    });
}) ;