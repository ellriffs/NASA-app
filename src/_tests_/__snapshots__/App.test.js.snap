// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App mounts App component to the virtual DOM 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <img
      alt="logo"
      class="Logo"
      src="nasa-worm-logo.png"
    />
    <form
      class="input-form"
    >
      <input
        class="Search"
        type="text"
      />
      <button
        class="submit"
        type="submit"
      >
        Search
      </button>
    </form>
    <p>
      No Results!
    </p>
  </div>
</DocumentFragment>
`;
